tpcc/pytpcc/runtime/loader.py

```diff
@@ -148,16 +148,23 @@ def loadWarehouse(self, w_id):
                 ## This is a new order: make one for it
                 if newOrder: no_tuples.append([o_id, d_id, w_id])
             ## FOR
-
-            self.handle.loadTuples(constants.TABLENAME_DISTRICT, d_tuples)
-            self.handle.loadTuples(constants.TABLENAME_CUSTOMER, c_tuples)
-            self.handle.loadTuples(constants.TABLENAME_ORDERS, o_tuples)
+            #
+            # print("Loading DISTRICT")
```

Comment:
Just added println to help us figure out where things get stick - I think it's ok?